Cargo.toml

```diff
@@ -0,0 +1,9 @@
+[workspace]
```

Comment:
I don't quite understand what you mean with the commit message. Could you update it?

I guess what you want to say is: setup the repository as a Rust workspace?
And that the members of the workspace are kvm-bindings and kvm-ioctls?

You also have a typo in "contralize"

Reply:
Exactly, I will update it